Lint-Whoppers/nutshell
# Jb factory function hope

# Description

I created a factory function with sams help yesterday. I now have it working and I altered my fetch calls and created another function with Rose and Sams help so that it now appends and displays on the Dom when new tasks are created.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions

1. Git fetch --all
2. git checkout jbFactoryFunctionHope
3. if you can see tasks on the dom then we are good!
4. Also there should be buttons set up for my next two features.



# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
